FhirToDataLake/data/schemas/r4/Patient.json

```diff
@@ -87,6 +87,51 @@
           "SubNodes": null,
           "ChoiceTypeNodes": null
         },
+        "_versionId": {
```

Comment:
Just wondering does it include extensions for all primitive values in the resource?

Reply:
yes, we generate primitive type extensions based on the [fhir shema](https://fhir-ru.github.io/downloads.html) without any filters, so I think it includes extensions for all the primitive values.

Is there any extension that is not listed in the schema?

Reply:
Got it. I didn't find anything left and just want to confirm whether I understand it correctly. Thanks!